ReleaseNotes-2.6.7

```diff
@@ -23,3 +23,5 @@ Bugfixes:
 * [webui] fix XSS attack vector via User.realname (bnc#950932)
 * [webui] fix XSS attack vector via Projec.title (bnc#950932)
 * [webui] add spec & changes file code highlighting
+* [webui] fix file save was not possible for files with code highlight (e.g. .js, .kiwi)
+* [webui] fix broken user show page 
```

Comment:
Fix order of packages/projects for the 'Involved Projects' table on the user home page